$NetBSD: patch-ab,v 1.1 2007/03/08 13:39:21 rillig Exp $

g++ 4.1.2 says:
SoExtSelection.cpp:351: error: extra qualification
	'SoExtSelectionP::SelectionState::' on member 'SelectionState'

--- src/nodes/SoExtSelection.cpp.orig	2005-06-09 13:35:44.000000000 +0200
+++ src/nodes/SoExtSelection.cpp	2007-03-08 14:36:41.000000000 +0100
@@ -348,7 +348,7 @@ public:
     SbList<SbVec2s> coords;
     SoTimerSensor * updatetimer;
 
-    SelectionState::SelectionState(SoExtSelection * t)
+    SelectionState(SoExtSelection * t)
     {
       this->updatetimer = new SoTimerSensor(&SoExtSelectionP::timercallback, t);
       this->updatetimer->setBaseTime(SbTime::zero());
@@ -356,7 +356,7 @@ public:
       this->reset();
     }
 
-    SelectionState::~SelectionState()
+    ~SelectionState()
     {
       delete this->updatetimer;
     }
